src/citrine/resources/project.py

```diff
@@ -199,6 +200,48 @@ def make_private(self,
         })
         return True
 
+    def list_members(self) -> List[User]:
+        """
+        List all of the users in the current project.
+
+        Returns
+        -------
+        List[User]
+            Dict with key 'users' containing an array of User objects
```

Comment:
I think it got swallowed up by your change, but I don't think this is returning the dict. Instead it just looks like it's returning the array of User objects.

Reply:
You are correct